### PR TITLE
Add HD path desc

### PR DESCRIPTION
**Pull request by Gustav-Simonsson**

### CODE REVIEW

File: source/includes/_wallet.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ HD Wallets can be created, deleted, and have new addresses generated. However, u
 
 When creating a wallet, one can optionally include one or more "subchain" indexes. These subchains can later be referenced when generating new addresses or sending txs. If none are provided in wallet creation, the wallet will derive & use addresses straight from the given extended pubkey. If no index is given when using the wallet with other APIs, it defaults to using the wallet's first (sub) chain.
 
+In BIP32 notation, the wallet layout is m/0, m/1, ... and m/<b>i</b>/0, m/<b>i</b>/1, ... for each subchain <b>i</b> if the wallet has subchains. For example, the path of the fourth address generated is m/3 for a non-subchain wallet. The path of the fourth address at subchain index two is m/2/3. Note that this is different from the default BIP32 wallet layout.
+
```
Comment (mflaxman) [inline source/includes/_wallet.md]: How about adding something like the following?:

For account 0 of a BIP32 default wallet layout, you'd submit the extended public key of m/0'/ (which can only be derived from your master private key) with `subchain indexes = [0, 1]`. Subchain index 0 represents the external chain (of account 0) and will discover all k keypairs that look like the following: m/0'/0/k. Subchain index 1 represents the internal chain (of account 0) and will discover all k keypairs that look like the following: m/0'/1/k.

https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#The_default_wallet_layout

For account 0 of a BIP 44 wallet (for BTC), you'd submit the extended public key of m/44'/0'/0'. (which can only be derived from your master private key) with `subchain indexes = [0, 1]`. Subchain index 0 represents the external chain (of account 0) and will discover all k keypairs that look like the following: m/44'/0'/0/0/k. Subchain index 1 represents the internal chain (of account 0) and will discover all k keypairs that look like the following: m/0'/1/k.

https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#examples

**Comment by Gustav-Simonsson** [comment]
@mflaxman Good suggestion, added your examples.

**Comment by Gustav-Simonsson** [comment]
@acityinohio Please check and feel free to edit anything you see can be improved.

**Comment by acityinohio** [comment]
Thanks @Gustav-Simonsson and @mflaxman, I'll take a looksie and merge soon.

**Pull request merged**